src/components/Article.jsx

```diff
@@ -1 +1,39 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
```

Comment:
this task is not fixed, please use the examples and recommendations above. If the test fails again, contact the chat in Slack with the error screen or come to the Q&A and we will analyze the task together

```suggestion
export const Article = ({ article }) => {
const { title, date, text } = article
```